src/rules/no-done-callback.ts

```diff
@@ -47,6 +46,7 @@ export default createRule({
     },
     schema: [],
     type: 'suggestion',
+    hasSuggestions: true,
```

Comment:
These will be removed once `main` is merged into `next` and my branch is rebased onto the new `next`